db/schema.rb

```diff
@@ -347,6 +347,7 @@
     t.datetime "created_at", null: false
     t.string "description"
     t.boolean "discoverable", default: false
+    t.string "invitation_slug"
```

Comment:
This should not be here, its from the chat channels changes PR

Reply:
Guess I got my wires crossed a bit 😄